core/src/main/java/com/linecorp/armeria/common/ResponseHeadersBuilder.java

```diff
@@ -42,6 +42,21 @@
      */
     ResponseHeadersBuilder status(HttpStatus status);
 
+    /**
+     * Sets the {@code "set-cookie"} header.\
```

Comment:
```suggestion
     * Sets the {@code "set-cookie"} header.
```
and add a link for `set-cookie`?